src/stylus/generic/_transitions.styl

```diff
@@ -107,17 +115,18 @@
     transform: translate(100%, 0)
 
 // Generic transitions
+.expand-transition
+  transition-default()
+
 .scale-transition
-  &-enter-active, &-leave-active
-    transition: $secondary-transition
+  transition-default()
```

Comment:
This was `$secondary-transition` transition before, dunno what the difference is.